book/quick_tour.md

```diff
@@ -1,53 +1,233 @@
+## Getting Help
 
-You can see that to make this work we didn't pass commandline arguments to [`ls`](/commands/docs/ls.md). Instead, we used the [`sort-by`](/commands/docs/sort-by.md) command that Nu provides to do the sorting of the output of the [`ls`](/commands/docs/ls.md) command. To see the biggest files on top, we also used [`reverse`](/commands/docs/reverse.md).
+Nushell provides an extensive, in-shell Help system. For example
 
-Nu provides many commands that can work on tables. For example, we could use [`where`](/commands/docs/where.md) to filter the contents of the [`ls`](/commands/docs/ls.md) table so that it only shows files over 1 kilobyte:
+```nu
+# help <command>
+help ls
+# Or
+ls --help
+# Also
+help operators
+help escapes
+```
 
-@[code](@snippets/introduction/ls_where_example.sh)
+::: tip Cool!
+Press the <kbd>F1</kbd> key to access the Help [menu](./line_editor.md#menus). Search for the `ps` command here, but _don't press <kbd>Enter</kbd> just yet_!
 
-Just as in the Unix philosophy, being able to have commands talk to each other gives us ways to mix-and-match in many different combinations. Let's look at a different command:
+Instead, press the <kbd>Down Arrow</kbd> key, and notice that you are scrolling through the Examples section. Highlight an example, _then_ press <kbd>Enter</kbd> and
+the example will be entered at the commandline, ready to run!
 
-@[code](@snippets/introduction/ps_example.sh)
+This can be a great way to explore and learn about the extensive set of Nushell commands.
+:::
 
-You may be familiar with the [`ps`](/commands/docs/ps.md) command if you've used Linux. With it, we can get a list of all the current processes that the system is running, what their status is, and what their name is. We can also see the CPU load for the processes.
+The Help system also has a "search" feature:
 
-What if we wanted to show the processes that were actively using the CPU? Just like we did with the [`ls`](/commands/docs/ls.md) command earlier, we can also work with the table that the [`ps`](/commands/docs/ps.md) command gives back to us:
+```nu
+help --find filesize
+# or
+help -f filesize
+```
 
-@[code](@snippets/introduction/ps_where_example.sh)
+It may not surprise you by now that the Help system itself is based on structured data! Notice that the output of `help -f filesize` is a table.
 
-So far, we've been using [`ls`](/commands/docs/ls.md) and [`ps`](/commands/docs/ps.md) to list files and processes in the form of a simple table. But data in Nu is structured and can be arbitrarily nested. As an example, let's now explore the [`help commands`](/commands/docs/help_commands.md) command.
+The Help for each command is stored as a record with:
 
-Running [`help commands`](/commands/docs/help_commands.md) gives us information for all Nushell commands as a table. However, the output will be quite large, so let's get the row for the `each` command only.
+- Its name
```

Comment:
"Its" shouldn't be on this line if it's also not on the other lines (maybe "Command name"?)

Reply:
Thanks - Fixed